src/Spinner.php

```diff
@@ -148,4 +145,16 @@ protected function eraseRenderedLines(): void
         $this->moveCursor(-999, -count($lines) + 1);
         $this->eraseDown();
     }
+
+    /**
+     * Clean up after the spinner.
+     */
+    public function __destruct()
+    {
+        parent::__destruct();
```

Comment:
While in `__constructs` the parent call should comes first, in `__desctruct`, the parent call should be the last line.

Reply:
TIL, thanks!